src/index.js

```diff
@@ -86,42 +78,75 @@ function activateCheckboxListeners() {
       const clickedCheck = e.target;
       const parent = clickedCheck.parentNode;
       const taskInput = parent.getElementsByClassName('task-text')[0];
+      const taskIndex = parent.getElementsByClassName('task-index')[0].value;
+      let status = false;
       if (clickedCheck.checked) {
         taskInput.classList.add('completed-task');
+        status = true;
+        updateTaskStatus(status, taskIndex, tasksLocal);
       } else {
+        status = false;
         taskInput.classList.remove('completed-task');
+        updateTaskStatus(status, taskIndex, tasksLocal);
       }
+      loadTasksFromLocalStorage();
+      const taskList = document.getElementById('task-list');
+      taskList.innerHTML = '';
+      /* eslint-disable no-use-before-define */
```

Comment:
- [ ] It's not a good approach to disable linters, fixing linters help us to write a clean and strong codebase 🙃 🙃